proposals/collection-expressions.md

```diff
@@ -188,23 +201,26 @@ namespace System.Runtime.CompilerServices
     }
 }
 ```
-
-The attribute can be applied to `class`, `struct`, and `interface` types.
+The attribute can be applied to a `class`, `struct`, `ref struct`, or `interface`.
 The attribute is not inherited although the attribute can be applied to a base `class` or an `abstract class`.
 
-An error is reported for a collection literal if there are multiple `[CollectionBuilder]` attributes on the collection type.
+The collection type must have an [*element type*](#element-type).
 
-The *create method* must be a `static` method defined on the *builder type*.
-The *create method* must have a single parameter of a type `System.ReadOnlySpan<T0>` passed by value, and a return value of the *collection type* to which the attribute is applied.
+For the *create method*:
+- The *builder type* must be a non-generic `class` or `struct`.
+- The method must be defined on the *builder type* directly.
```

Comment:
maybe clarify that that means it cannot appear in a base type? (just a nit, this is probably sufficient).